src/librustc_borrowck/lib.rs

```diff
@@ -43,7 +43,8 @@ extern crate core; // for NonZero
 
 pub use borrowck::check_crate;
 pub use borrowck::build_borrowck_dataflow_data_for_fn;
-pub use borrowck::{AnalysisData, BorrowckCtxt, ElaborateDrops};
+pub use borrowck::{AnalysisData, AnalysisResult, BorrowckCtxt, ElaborateDrops, Loan, SafeLoan};
```

Comment:
This is tied to the comments I made regarding the use of `tcx.borrowck`. In order to support the MIR version of borrowck, should I hide these again and instead expose the `rls-data` structs?